containers/runner/launch

```diff
@@ -112,11 +115,16 @@ elif [ ${TEST_JOBS} -gt 4 ]; then
     VAR_TMP="-v /var/tmp"
 fi
 
+if $CRUN -v 2>&1 | grep -q "podman" && [ $(id -u) -ne 0 ]; then
```

Comment:
A cheaper way to do this is `if [ "${CRUN%podman*}" != "$CRUN" ] ...`, avoids actually calling the program. Not a biggie, just a generally good style in shell scripts where performance matters a bit more.

Reply:
These variables changing and comparing to itself is something totally unreadable. However, yes it's good style and I should learn to use it :). Thanks for idea.

Reply:
Ohh this will not work because it's not podman but ``/usr/bin/podman`` and I don't want to test the complete path in case it will not be like that in some situations.

Reply:
Yes, it's a bit unreadable if you don't know it indeed.. POSIX shell string manipulation :-/ As I said, that's only nitpicking, it's ok like this if you prefer. Otherwise, `${CRUN%podman}` is a bit cleaner indeed in this case (no `*`)

Reply:
My bad it's working correctly.